src/extension/wps/src/test/java/au/org/emii/download/ParallelDownloadManagerTest.java

```diff
@@ -1,4 +1,4 @@
-package au.org.emii.gogoduck.download;
```

Comment:
Line 233  function toSet can be replaced by 

`Set<T> testRequestsSet = new HashSet<DownloadRequest>(Arrays.asList(testRequests));`